packages/api/src/routes/feedback.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { z } from "zod";
+import { getFeedbackOrFail } from "../command/feedback/feedback";
+import { createFeedbackEntry } from "../command/feedback/feedback-entry";
+import { requestLogger } from "./helpers/request-logger";
+import { asyncHandler, getFromParamsOrFail } from "./util";
+
+const router = Router();
+
+/** ---------------------------------------------------------------------------
+ * GET /feedback/:id
+ *
+ * Returns the id and status of the feedback.
+ * Mostly to validate it exists.
+ */
+router.get(
+  "/:id",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const id = getFromParamsOrFail("id", req);
+    await getFeedbackOrFail({ id });
+    // TODO add status to the feedback and return it here
```

Comment:
I didn't add status because I didn't want to make it too complicated on this PR and not sure whether we need that or not. So might as well just remove this and the logic on the API contract, plus the upstream Dash.

Reply:
done